apis/kueue/v1beta1/workload_types.go

```diff
@@ -104,6 +104,14 @@ type PodSetTopologyRequest struct {
 	// +optional
 	Preferred *string `json:"preferred,omitempty"`
 
+	// unconstrained indicates that Kueue has freedom to schedule the PodSet within
```

Comment:
With preferred Kueue also has this freedom.  Pivot the description more toward lack of need or preference for any specific placement.

Reply:
Done